terms/context.md

```diff
@@ -1,2 +1,5 @@
 # Context
 
```

Comment:
Each term entry needs to start with an explicit one-sentence definition of what the term is. This entry is useful, just needs that extra term.

(note that I threw a lot of things in there from the wildly unstructured google doc so I'm still working on bringing a lot of existing terms up to this standard, as well as adding a style guide in the contributing section to say what I'm doing).

Reply:
OK. I don't think my suggestion is optimal from this perspective. I think the reason that we have this page is because contextual metadata means something very specific and context means something very broad. Is might be better to dedicate this page to contextual metadata rather than context.